app/src/services/object.js

```diff
@@ -34,7 +34,8 @@ const service = {
         public: data.public,
         active: data.active,
         bucketId: data.bucketId,
-        createdBy: data.userId ?? SYSTEM_USER
+        createdBy: data.userId ?? SYSTEM_USER,
+        lastSyncedDate: new Date()
```

Comment:
This line is disingenuous. The intent of the lastSyncedDate is to record the last time an _explicit_ synchronization operation was applied to the object. When an object is freshly created via COMS, while it is technically synchronized with S3 as it was COMS that pushed up the new content, there was no actual synchronization operation performed. We should not be populating it here.